network/transport/types.go

```diff
@@ -94,6 +116,8 @@ type Diagnostics struct {
 	// SoftwareID contains an indication of the vendor of the software of the node. For open source implementations it's recommended to specify URL to the public, open source repository.
 	// Proprietary implementations could specify the product's or vendor's name.
 	SoftwareID string `json:"softwareID"`
+	// Certificate contains the certificate presented by the peer during the TLS handshake.
+	Certificate string `json:"certificate"`
```

Comment:
could've been OK to just return it as `[]byte` (meaning DER, no need to PEM encode?)

But PEM is probably more recognizable, so OK.

Reply:
PEM is readable when printed, makes my life a bit easier